components/nexus-core/src/main/java/org/sonatype/nexus/internal/httpclient/DefaultsCustomizer.java

```diff
@@ -58,6 +61,9 @@ public DefaultsCustomizer(
     this.requestTimeout = checkNotNull(requestTimeout);
     log.debug("Request timeout: {}", requestTimeout);
 
+    this.connectionRequestTimeout = checkNotNull(connectionRequestTimeout);
+    log.debug("Connection request timeout: {}", connectionRequestTimeout);
```

Comment:
a single log line for all of the params seems more appropriate. ( toString? ). Imagine these sparse log lines spread across other thread logging in a debug log is not very tidy and harder to grep.